src/DSharp.Mscorlib/Collections/Generic/Dictionary.cs

```diff
@@ -8,44 +8,57 @@ namespace System.Collections.Generic
     [ScriptIgnoreNamespace]
     [ScriptImport]
     [ScriptName("Object")]
-    public sealed class Dictionary<TKey, TValue> : IEnumerable<KeyValuePair<TKey, TValue>>
+    public sealed partial class Dictionary<TKey, TValue>
+        : IDictionary<TKey, TValue>
```

Comment:
I Wouldn't normally start a new line with commas or other seperator tokens personally.

Reply:
It's all about version controlling, this is usually the recommended way to avoid conflicts but I'm open for suggestions

Reply:
Not sure how this improves it honestly, but as we have no standards for DSharp it's a bit of a mute point ;) Lets leave it for now because it's not a huge deal :)